src/js/components/navbars.js

```diff
@@ -85,48 +86,127 @@
 
   var module = angular.module('mobile-angular-ui.components.navbars', []);
 
-  /**
-   * @directive navbarAbsoluteTop
-   * @restrict C
-   * @description
-   *
-   * Setup absolute positioned top navbar.
-   *
-   * ``` html
-   *  <div class="navbar navbar-app navbar-absolute-top">
-   *    <!-- ... -->
-   *  </div>
-   * ```
-   */
+  var global;
```

Comment:
That's quite too generic to be understood. What is it supposed to hold, why u need a singleton for that?

I've still to read the rest but looks like a disruptive approach. We'll fix it anyway i like your idea of supporting common patterns.